src/main/kotlin/conduit/config/AppConfig.kt

```diff
@@ -1,12 +1,26 @@
 package conduit.config
 
+import io.jsonwebtoken.SignatureAlgorithm
+import org.http4k.filter.CorsPolicy
+import javax.crypto.spec.SecretKeySpec
+
 data class AppConfig(
     val logConfig: String,
     val db: DbConfig,
+    val corsPolicy: CorsPolicy,
+    val jwtConfig: JwtConfig,
     val port: Int
 )
 
 data class DbConfig(
     val url: String,
     val driver: String
 )
+
+data class JwtConfig(
+    private val secret: String = System.getenv("JWT_SIGNING_KEY_SECRET"),
+    private val algorithm: String = System.getenv("JWT_SIGNING_KEY_SECRET") ?: SignatureAlgorithm.HS256.jcaName,
```

Comment:
They are used immediately on the next line where the signingKey is generated and then never needed again.  I was hoping it might lead to some security advantage.  

http4k goes even further with secrets that are destroyed after use but I didn't think the scope of this PR should try to change out the environment variable handling.  Not to mention that I'm a newbie at http4k.

https://www.http4k.org/guide/modules/cloud_native/